scripts/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+./scripts/feeds update -a
+./scripts/feeds install -a
+./scripts/feeds update packages
+./scripts/feeds install -a -p packages
+cp insp_config .config
+make menuconfig
```

Comment:
@kollya should we every time install the packages. It is a one time thing correct?

Also, make -j<no_depends_on_the_core> is missing

Reply:
@gauthamkantharaju install packages command is fast, no problem if they are already installed. 

the last commit adds script param for make -jN

Reply:
@kollya after make menuconfig it should be make -j $1 correct?